### PR TITLE
Dashboard report statistics

### PR DESCRIPTION
**Pull request by jaleszek**
Dashboard statistics for the Admin.

### CODE REVIEW

File: app/models/form_answer_statistics/picker.rb [hunk 1]
```diff
@@ -0,0 +1,117 @@
+class FormAnswerStatistics::Picker
+  def registered_users
```
Comment (matthewford) [inline app/models/form_answer_statistics/picker.rb]: Assignment Branch Condition size for registered_users is too high. [16.03/15]

File: app/models/form_answer_statistics/picker.rb [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    out = {}
+    out[:last_24h] = User.where(created_at: (Time.now - 24.hours)..Time.now).count
+    out[:last_7_days] = User.where(created_at: (Time.now - 7.days)..Time.now).count
+    out[:total_so_far] = User.count
+    out
+  end
+
+  def applications_table
```
Comment (matthewford) [inline app/models/form_answer_statistics/picker.rb]: Assignment Branch Condition size for applications_table is too high. [51.24/15]

File: app/models/form_answer_statistics/picker.rb [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+  def collect_submission_ranges(scope)
+    temp = []
+    temp << submissions_query(scope, DateTime.now - 1.day)
+    temp << submissions_query(scope, DateTime.now - 7.days)
+    temp << submissions_query(scope, nil)
+    temp
+  end
+
+  def collect_completion_ranges(scope)
```
Comment (matthewford) [inline app/models/form_answer_statistics/picker.rb]: Assignment Branch Condition size for collect_completion_ranges is too high. [26.93/15]

File: app/models/form_answer_statistics/picker.rb [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+  end
+
+  def applications_completions
+    # TODO: SCOPE WITH YEAR
+    out = {}
+    klass::POSSIBLE_AWARDS.each do |aw|
+      scope = klass.where(award_type: aw)
+      out[aw] = collect_completion_ranges(scope)
+    end
+    out["total"] = collect_completion_ranges(klass.where.not(state: "not_eligible").where(submitted: false))
```
Comment (matthewford) [inline app/models/form_answer_statistics/picker.rb]: Line is too long. [108/100]

File: app/models/form_answer_statistics/picker.rb [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+  def applications_table
+    out = {}
+    registered_users = []
+    registered_users << User.where(created_at: (Time.now - 24.hours)..Time.now).count
+    registered_users << User.where(created_at: (Time.now - 7.days)..Time.now).count
+    registered_users << User.count
+    out[:registered_users] = { name: "Registered Users", counters: registered_users }
+
+
```
Comment (matthewford) [inline app/models/form_answer_statistics/picker.rb]: Extra blank line detected.

**Comment by RuslanHamidullin** [comment]
Good job I think :+1:  
But looks like travis is down :boar:

**Pull request merged**